### DISCUSSION TITLE
ViGEmBus EOL

### DISCUSSION THREAD

**Discussion by eTaurus** [LizardByte/community#335] [Sunshine]
Hi!
How is the End-Of-Life of ViGEmBus going to affect Sunshine?

https://docs.nefarius.at/projects/ViGEm/End-of-Life/

**Comment by ReenigneArcher** [comment]
At this point, we're not entirely sure.

One idea is for us to pick up the maintenance of these, although it's not entirely desirable.

I have forked some of the vigem repos in preparation for this. https://github.com/orgs/LizardByte/repositories?q=virtual&type=all&language=&sort=

The driver signing is the annoying thing we would have to deal with.

**Reply by eTaurus** [reply]
Thank you for your reply. The main reason I started this conversation was to check if you know of this problem. I'm no developer so I do not understand the scope of the consequences Nefarius' decision has.

[a comment by an automated account is not shown]
